sleap/gui/color.py

```diff
@@ -106,6 +115,14 @@ def tracks(self) -> Iterable[Track]:
         if self.labels:
             return self.labels.tracks
         return []
+    
+    @property
+    def instanceGroups(self) -> Iterable[InstanceGroup]:
+        """Gets instance groups for project."""
+        if (self.session and self.frame_idx):
+            return self.session.frame_groups[self.frame_idx].instance_groups
```

Comment:
This could error since there may not yet be a `FrameGroup` at the current `self.frame_idx`. A safer way to do this might be:
```suggestion
            frame_group = self.session.frame_groups.get(self.frame_idx, None)
            if frame_group is not None:
                return frame_group.instance_groups
```